sources/packages/backend/libs/services/src/students-assessments/assessment-sequential-processing.service.ts

```diff
@@ -343,4 +385,139 @@ export class AssessmentSequentialProcessingService {
       options?.alternativeReferenceDate,
     );
   }
+
+  /**
+   * Get SFAS application awards totals.
+   * @param assessment current assessment being processed.
+   * @param firstAssessmentDate date of the first assessment date of the current application.
+   * @returns SFAS application awards totals.
+   */
+  async getProgramYearSFASAwardsTotals(
+    assessment: StudentAssessment,
+    firstAssessmentDate: string,
+  ): Promise<AwardTotal[]> {
+    const lastName = assessment.application.student.user.lastName;
+    const sin = assessment.application.student.sinValidation.sin;
+    const birthDate = assessment.application.student.birthDate;
+    const programYearStartDate = assessment.application.programYear.startDate;
+    const sfasApplicationAwards = this.sfasApplicationsRepo
+      .createQueryBuilder("sfasApplication")
+      .select("SUM(sfasApplication.csgp_award)", "CSGP")
```

Comment:
Unless specific scenarios where not possible we should try to use as much as possible the entity model properties, for instance, instead of the below.
```
.select("SUM(sfasApplication.csgp_award)", "CSGP")
```
We can try to use.
```ts
.select("SUM(sfasApplication.csgpAward)", "CSGP")
```